PWGJE/FlavourJetTasks/AliAnalysisTaskHFJetIPQA.h

```diff
@@ -3,6 +3,7 @@
 #include "AliAnalysisTaskEmcalJet.h"
 #include "TGraph.h"
 #include "THistManager.h"
+#include "THnSparse.h"
```

Comment:
This is not necessary, you still use a pointer type, it could be handled with a forward declaration. Better stay with forward declarations in the header file unless including is absolutely needed (i.e. because you used member functions in the header or you use objects as value types).